test/task.h

```diff
@@ -184,6 +184,7 @@ enum test_status {
 #  define inline __inline
 # endif
 
+# if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
@equalsraf, I just tested it and it turned out that ifndef doesn't fly with snprintf's body definition. So the slim version of your initial check would.

```c
# if _MSC_VER < 1900
```

(is `defined(_MSC_VER) && ` part required in that scope?)

Secondly, for sake if completion of this PR, please apply this patch as well: https://gist.github.com/jasonwilliams200OK/36974301200f2eff5bbc.

After this, it definitely compiles libuv with VS2015 RC! :sunglasses: :clap: